pkg/cmd/server/admin/create_bootstrappolicy_file.go

```diff
@@ -16,12 +16,14 @@ import (
 
 	"github.com/openshift/origin/pkg/api/latest"
 	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/config/defaults"
```

Comment:
This is somewhat questionable - while DefaultPolicyFile was wierd before, this should really just be policy.json.  Don't try to overthink it.